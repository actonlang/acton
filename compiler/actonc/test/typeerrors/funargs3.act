def f(x,y):
    return x+y

a = f(x=3,4)
