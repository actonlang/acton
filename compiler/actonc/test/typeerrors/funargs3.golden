Building file test/typeerrors/funargs3.act using temporary scratch directory
[error Parse error]: unexpected '4'
                     expecting "**" or closing ')'

     +--> test/typeerrors/funargs3.act@4:11-4:12
     |
   4 | a = f(x=3,4)
     :           ^ 
     :           `- unexpected '4'
     :              expecting "**" or closing ')'
     :              
-----+
